blaze/server/serialization.py

```diff
@@ -34,6 +97,132 @@ def _coerce_str(bytes_or_str):
 )
 
 
+try:
+    import blosc
+    del blosc
+    compress = 'blosc'
+except ImportError:
+    compress = None
+
+
+def _final_base(values):
+    base = values.base
+    while True:
+        try:
+            base = base.base
+        except AttributeError:
```

Comment:
when the base is a buffer then the buffer passed will be the base.

For example:
```python
In [1]: np.frombuffer(b'abc', dtype='uint8').base
Out[1]: b'abc'
```